cmd/ttn-lw-stack/commands/start.go

```diff
@@ -228,6 +228,9 @@ var startCommand = &cobra.Command{
 
 		if start.Console || startDefault {
 			logger.Info("Setting up Console")
+			if config.Console.UI.TemplateData.SentryDSN == "" {
```

Comment:
I don't think this if-statement is necessary since the `SentryDSN` value cannot be set explicitly.

Reply:
The config should also be added to the oauth app.

Reply:
@htdvisser Do we want to enable sentry for the web UIs by default, when a global sentry DSN config is set? Implemented like that, there would be no way to opt-out.

Reply:
I think that is fine. If a deployment wants to enable error reporting, that includes both frontend and backend errors. They may choose to use a different DSN for frontend, which is possible with the current implementation.